tests/test_json_schema_artifact.py

```diff
@@ -0,0 +1,29 @@
+import json
+import pkgutil
+import unittest
+import fastjsonschema
+
+from nmdc_schema import nmdc_data
+
+
+class TestJsonSchemaArtifact(unittest.TestCase):
+    def test_artifact_compiles_with_fastjsonschema(self):
+        """Test that the NMDC JSON Schema compiles with fastjsonschema
+
+        This is an important use case because nmdc-runtime uses fastjsonschema to validate data
+        against the schema. fastjsonschema has slightly different behavior than the jsonschema,
+        which underlies the linkml-validate command.
+        """
+        json_schema = nmdc_data.get_nmdc_jsonschema_dict()
+        validate = fastjsonschema.compile(json_schema)
+        data = validate({})
+        assert data == {}
+
+    def test_materialized_pattern_artifact_compiles_with_fastjsonschema(self):
+        """Test that the NMDC JSON Schema with materialized patterns compiles with fastjsonschema"""
+        json_schema_bytes = pkgutil.get_data("nmdc_schema",
+                                             "nmdc_materialized_patterns.schema.json")
+        json_schema = json.loads(json_schema_bytes)
+        validate = fastjsonschema.compile(json_schema)
+        data = validate({})
+        assert data == {}
```

Comment:
Props to whoever wrote these approachable docstrings! 👏

Reply:
@eecavanna at least in this branch, I am credited as the author of this entire file. I really don't think that's the case. Can you please help me figure out what happened?

Reply:
According to Git, the file was created by you in this commit: https://github.com/microbiomedata/berkeley-schema-fy24/commit/e35178672aa559d4bfb8d24ee188e79bfde6d48c#diff-d66bad7d4246ba73f4053e7a121053a4c7898d20c631961595cc616d4077b75a

That is in contrast to Git saying that it existed before that commit.

I'll dig into this a little bit to come up with a hypothesis.

Reply:
My guess is that _part_ of this merge was done by copy/pasting files or their content from one branch to the other, instead of using Git's own merge functionality. I don't know how likely that is, given the specific merge workflow you are using. 🤷

Reply:
thanks. if there's a way to restore credit/blame to @pkalita-lbl , I'd like to do that. I'm pretty sure I didn't edit the file.